<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgExportWindow</class>
 <widget class="QDialog" name="ImgExportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Format</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="bootCfgGroupBox">
     <property name="title">
      <string>File Format</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="f0RadioButton">
          <property name="text">
           <string>png</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f1RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>YV12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f2RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>YU12/I420</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f3RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>NV21</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f4RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>NV12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f5RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>YUY2/YUYV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f6RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>YVYU</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f7RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>UYVY</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f8RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>4:4:4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f9RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>RGB565_L</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="f10RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>RGB565_B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f11RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>BGR565_L</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f12RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>BGR565_B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f13RadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>RGB888</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f14RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerBG</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f15RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerGB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f16RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerRG</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f17RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerGR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f18RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerBG_RAW10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f19RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerGB_RAW10</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="f20RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerRG_RAW10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f21RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerGR_RAW10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f22RadioButton">
          <property name="text">
           <string>BayerBG_RAW12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f23RadioButton">
          <property name="text">
           <string>BayerGB_RAW12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f24RadioButton">
          <property name="text">
           <string>BayerRG_RAW12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f25RadioButton">
          <property name="text">
           <string>BayerGR_RAW12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f26RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerBG_RAW16</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f27RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerGB_RAW16</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f28RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerRG_RAW16</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="f29RadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>BayerGR_RAW16</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImgExportWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImgExportWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
